server/data_access_layer/user.js

```diff
@@ -0,0 +1,22 @@
+const { base } = require('./index');
+const { extractContentFromRecords, getAllFromTable } = require('./helpers');
+
+async function getUser(userID) {
+  const userTable = base('Users').select({
+    // Filter by user id
+    filterByFormula: '{ID}=' + userID,
+    view: 'Grid view'
+  });
+
+  const user = await getAllFromTable(userTable);
+  var userData = {
+    'Full Name': user[0]['fields']['Full Name'], 'Email': user[0]['fields']['Email'],
+    'Current Title': user[0]['fields']['Current Title'], 'Answered': user[0]['fields']['Answered']
+  };

```

Comment:
optional:

```
const user = await getAllFromTable(userTable)[0];
const userData = {
     'Full Name': user['fields']['Full Name'], 'Email': user['fields']['Email'],
      'Current Title': user['fields']['Current Title'], 'Answered': user['fields']['Answered']
    };
```

Reply:
agreed ^ 
+ can you also split each field onto a newline so it's easier to read